ciongke/cmd/test-pr/Dockerfile

```diff
@@ -0,0 +1,9 @@
+FROM alpine:3.4
```

Comment:
Please add copyright info to all Dockerfiles: https://github.com/kubernetes/kubernetes/blob/master/hack/boilerplate/boilerplate.Dockerfile.txt

Also please set `MAINTAINER` https://docs.docker.com/engine/reference/builder/#/maintainer

And a description... maybe `LABEL description=foo` https://docs.docker.com/engine/reference/builder/#/label